src/components/note/note.component.tsx

```diff
@@ -88,21 +89,23 @@ export const Note = ({
   return (
     <EditorContext.Provider value={{ onLinkAdded, editMode: false }}>
       <StyledNote width={width} {...rest} data-component="note">
-        {title && (
-          <StyledTitle hasInlineControl={!!inlineControl}>{title}</StyledTitle>
-        )}
-
-        {inlineControl && (
-          <StyledInlineControl>{inlineControl}</StyledInlineControl>
-        )}
-
-        <StyledNoteContent hasInlineControl={!!inlineControl}>
-          <Editor
-            readOnly
-            editorState={getDecoratedValue(noteContent)}
-            onChange={/* istanbul ignore next */ () => {}}
-          />
-        </StyledNoteContent>
+        <StyledNoteMain>
+          <StyledNoteContent hasInlineControl={!!inlineControl}>
+            {title && (
+              <StyledTitle hasInlineControl={!!inlineControl}>
```

Comment:
```suggestion
              <StyledTitle>
```

Reply:
There's also another instance in this file where we pass `hasInlineControl`, it doesn't look like it's used anymore in the style file so it can be removed